### PR TITLE
SYMP_AVE_HACK

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@drealeed Please Review

**Comment by jchambers-ln** [comment]
Code Looks good to me -- will need to test to be sure it does the trick.

File: widgets/src/marshaller/HipieDDL.js [hunk 1]
```diff
@@ -60,6 +60,17 @@
...
+                if (val === undefined && rhsKey.indexOf("_AVE") === rhsKey.length - 4 && item.base_count !== undefined) {
+                    var rhsSum = rhsKey.substring(0, rhsKey.length - 4) + "_SUM";
+                    val = item[rhsSum];
+                    if (val === undefined) {
+                        val = item[rhsSum.toLowerCase()];
+                    }
+                    if (val) {
+                        val /= item.base_count;
+                    }
+                }
```
Comment (drealeed) [inline widgets/src/marshaller/HipieDDL.js]: will this handle summing multiple avges and dividing it by the sum of multiple base_counts for aggregated wuid-based charts?

**Comment by drealeed** [comment]
If the answer to the above question is yes, reviewed & signed off

**Comment by jchambers-ln** [comment]
I tested this in DSP and it seems to work rendered pie, line, choropleth using it with good results.

File: widgets/src/marshaller/HipieDDL.js (hunk 1 above)
Comment (GordonSmith) [inline widgets/src/marshaller/HipieDDL.js]: I don't think so.  

This is simply taking the the SUM and dividing by the base_count on a per row basis.

In the example I looked at the aggregate (SUM) had already been burned into the WU Result?

**Comment by GordonSmith** [comment]
@jchambers-ln Where the number correct though?

**Comment by jchambers-ln** [comment]
I did a few spot checks and assuming hippie is calculating Sum and count correctly the math to get the average is correct.

**Pull request merged**